docs/parameters.md

```diff
@@ -10,6 +10,7 @@ Below is the set of supported parameters accepted by the pipeline.
 | revision            | Revision of the Source Repository. Typically use `{{ revision }}`.                   |                                     |
 | target-branch       | What is the package branch we work with, e.g., `rhel-10-main`, `{{ target_branch }}` |                                     |
 | build-architectures | Array of architectures we want to build for                                          | `[aarch64, s390x, ppc64le, x86_64]` |
+| build-platforms     | Array of platform overrides (MPLs)                                                   | []                                  |
```

Comment:
Which platforms?  What is MPLs?  If set, you should document which multi platform controller tasks are affected by this?